tests/foreman/cli/test_contentcredentials.py

```diff
@@ -45,6 +45,27 @@ def create_gpg_key_file(content=None):
     return None
 
 
+def wait_for_repo_metadata_tasks(sat, org_name, repo_name='', product_name=''):
+    """Search and wait for any repository metadata generate task(s).
+
```

Comment:
Can you please explain the rationale for adding a new function for this instead using existing `wait_for_tasks` helper func directly in tests?

Reply:
They all need pretty much the same thing so saw the opportunity to streamline, 
and perhaps any future use in other CLI tests?